services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -471,10 +471,18 @@ class AppSettings(BaseCustomSettings, MixinLoggingSettings):
     )
 
     # for passing self-signed certificate to spawned services
-    # TODO: fix these variables once the timeout-minutes: 30 is able to start dynamic services
-    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_ID: str = ""
-    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME: str = ""
-    DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME: str = ""
+    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_ID: Optional[str] = Field(
+        None,
+        description="ID of the docker secret containing the self-signed certificate",
+    )
+    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME: Optional[str] = Field(
+        None,
+        description="Name of the docker secret containing the self-signed certificate",
+    )
+    DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME: Optional[str] = Field(
+        None,
+        description="Filepath to self-signed osparc.crt file *as mounted inside the container*",
+    )
```

Comment:
I would change it like this
```suggestion
    DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME: str = Field(
        "",
        description="Filepath to self-signed osparc.crt file *as mounted inside the container*, empty strings disables it",
    )
```